### ISSUE TITLE
Can we set the checkbox by code?

### ISSUE THREAD

**Issue by putrarp** [TheJuki/KFormMaster#169]
i try to change unchecked checkbox to be checked by using below code

```
val quot = formBuilder.getFormElement<FormCheckBoxElement<Boolean>>(Tag.Quotable.ordinal)
quot.value = true
//or using
quot.setValue(true)
```

the value is changed to true but the checkbox remains unchecked.
is there any way to change the checkbox to become checked?

**Comment by TheJuki** [comment]
I just released [6.5.0](https://github.com/TheJuki/KFormMaster/releases/latest). Please update your dependency and let me know if setting the value updates the checkbox on the form.

**Comment by putrarp** [comment]
It's working like a charm! Thanks for the update!  :)
